Packages/MIES/MIES_NeuroDataWithoutBorders.ipf

```diff
@@ -941,6 +941,10 @@ static Function NWB_ClearWriteChannelParams(s)
 	samplingRate = s.samplingRate
 	groupIndex   = s.groupIndex
 
+	// Clear wave elements before overwriting with a default initialized structure.
+	// This avoids a memory leak. (Reported to WM)
```

Comment:
@t-b I'd like to your thought on this when we chat next.

Reply:
Sure!